addons/api/mirage/scenarios/ipc.js

```diff
@@ -172,10 +172,12 @@ export default function initializeMockIPC(server, config) {
     /**
      * Do nothing when attempting to minimize, toggle fullscreen,
      * and close a browser window
+     * * Do nothing when canceling a session invokes the stop method
      */
     minimizeWindow() {}
     closeWindow() {}
     toggleFullscreenWindow() {}
+    stop() {}
```

Comment:
This was added because cancelSession actions in desktop ui using `this.ipc.invoke('stop, {...}` were causing an error due to `stop()` not being defined. This error prevented the success toast from displaying. This is only an issue in mirage. 
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/73a4a8ec-4378-47b8-ae18-bb7275d9e9fd)